tox.ini

```diff
@@ -56,11 +56,13 @@ deps =
     {format,checkformatting}: -r requirements/format.txt
     lint: -r requirements/lint.txt
     tests: -r requirements/tests.txt
-    coverage: -r requirements/tests.txt
+    coverage: -r requirements/coverage.txt
```

Comment:
Changed the `coverage` environment to use its own requirements file since it only needs to install `coverage` not all of the test dependencies.